google/resource_bigquery_dataset.go

```diff
@@ -72,7 +72,7 @@ func resourceBigQueryDataset() *schema.Resource {
 				Optional:     true,
 				ForceNew:     true,
 				Default:      "US",
-				ValidateFunc: validation.StringInSlice([]string{"US", "EU"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"US", "EU", "asia-northeast1"}, false),
```

Comment:
Can you remove the possible values from the comment at line 68. The comment is stale and the ValidateFunc makes it obvious what are the valid values. Same applies for the bit in the comment talking about the default value.

Reply:
Done!